src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestOptionsKey.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System.Net.Http
+{
+    public readonly struct HttpRequestOptionsKey<TValue>
+    {
+        private readonly string _key;
```

Comment:
You can remove this and then use the:
public string Key { get; }
syntax for a get-only autoprop that the ctor assigns to.

Reply:
Addressed in recent commits